broadinstitute/single_cell_portal_core
# Reuse validation message component in upload UI, refine error message (SCP-4119)

This refactors `FileUploadControl.js` to use `ValidationMessage`, the newly resurrected component used in the sync UI.

This improves maintainability by replacing duplicate code with a more general approach.  It also fixes something of a bug in an error message, where the shown message could be much too long.

To test:
* In upload UI, select errant file
* Confirm that error message includes a note on how to email support, including the study's accession

This satisfies SCP-4119.
